### DISCUSSION TITLE
docs/ninjutsu/low-ranking-ninja/take-a-look-site

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#9] [General]
# docs/ninjutsu/low-ranking-ninja/take-a-look-site

狂暴冲浪模式启动！ 一头扎进互联网的海底。

https://996.ninja/docs/ninjutsu/low-ranking-ninja/take-a-look-site

**Comment by BBFTZ** [comment]
6666

**Comment by Endsoon** [comment]
那个spicy chat很容易就摸到记忆上限导致AI失忆，后面发现个同质网站JanitorAI感觉更好一点
“https://janitorai.com”

**Reply by Endsoon** [reply]
https://janitorai.com

**Reply by MingGH** [reply]
好！我看看

**Reply by MingGH** [reply]
加上了，周末再一起放出来了

**Reply by MingGH** [reply]
非常好玩滴干活，写出来了[上忍 | Janitorai AI-角色扮演](https://996.ninja/docs/ninjutsu/up-ranking-ninja/AI-can-make-you-a-legend)

**Comment by lg961126** [comment]
出错了: Network Error 
orz

**Reply by MingGH** [reply]
没有哇

**Comment by Shawyuan2022** [comment]
终于回来了，消失两个多月了，QAQ急死我了！

**Comment by Moca-H** [comment]
http://tinywaste.maougame.com/ 这个文字游戏 到底怎么使用炊具啊？ 卡起了

**Comment by Dwizeed** [comment]
wtf,win93上的那个trollbox里面还有真人？？？

**Reply by MingGH** [reply]
确实是真人，我去！
![](https://img.996.ninja/ninjutsu/7eb54b6f9643b0ecf0887353bfd58067.png)

**Comment by pairusuo** [comment]
我来提供一个奇怪的网站
https://touchfish.click

**Reply by MingGH** [reply]
好好好，感谢！
